helm/cluster-openstack/templates/_helpers.tpl

```diff
@@ -39,6 +39,15 @@ giantswarm.io/organization: {{ .Values.organization | quote }}
 application.giantswarm.io/team: {{ index .Chart.Annotations "application.giantswarm.io/team" | quote }}
 {{- end -}}
 
+
+{{/*
+Selector labels
```

Comment:
I am totally OK with renaming this as labels.selector. It is definitely better than ours. Please fix it for CAPVCD too.